tests/opentracer/test_tracer.py

```diff
@@ -577,3 +577,15 @@ def test_required_dd_fields(self):
         with tracer.start_span('my_span') as span:
             assert span._dd_span.service
 
+
+def test_set_global_tracer():
```

Comment:
That works, but we need to update the `set_global_tracer()` so that it syncs also the `ddtrace.tracer`.